CHANGELOG-1.3.md

```diff
@@ -18,6 +18,7 @@ and date `## vX.Y.Z - YYYY-MM-DD` and create a new placeholder section for  `unr
 
 ## unreleased
 
+* [ENHANCEMENT] Allow configuring the namespace of service monitors
```

Comment:
Nit: enhancements should come after changes. Also, I wonder if we need to include the GH issue number and/or Jira ticket number.

Reply:
I actually thought we were adding updates in reverse chronological order 🤔  aren't we?

Reply:
My understanding is that we have 2 ordering keys: by type first, then by reverse chronological order.

Reply:
Scratch that, I just read the header section for the first time with the right level of attention 😅